app/views/content/help-and-advice.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Help and advice
+navigation: 50
+---
+
+We’re here to answer your questions and provide free advice about teacher training, whether you’re just thinking about teaching or you’re ready to apply.
+
+##Come to an event
+
+Our events are a great way to meet people and talk about any concerns or questions you have about becoming a teacher.
+
+###Train to Teach
+
+Train to Teach events are free to attend and you'll be able to:
+
+- put your questions to expert advisers, teachers and training providers
+- chat with current teachers
+- find out what a career in teaching is really like
+- watch presentations which provide detailed guidance on how to get into teaching, the application process and funding your training
+
+
+###Online question and answer sessions
+
+There are online question and answer sessions where you can ask a panel of specialists the questions that matter to you. These sessions cover a range of different topics so you can ask specific questions and receive advice tailored to your particular circumstances.
+
+###School and university events
+
+You could also attend one of the School and University events, either online or in person, and hear directly from teacher training providers about the courses they offer.
+
+Search for an event near you (THIS IS A BUTTON)
```

Comment:
I don't think we should say 'near you' as that implies they are all in person?

When we have the new events design, we can be smarter about linking to pre-filtered results (eg 'Search for an online event' or 'Enter your postcode...') but not yet!